report/main.qmd

```diff
@@ -0,0 +1,139 @@
+
+# Exploratory Analysis
+
+Given that the data consists of a time series of sales across three sites, it was necessary to carry out a time-based analysis. In that sense, @fig-1 helps visualize the beverages sold and the shadows behind the lines display the distinct intervention periods by site. Additionally, @sec-appeda summarizes the effect from past observations in the new data points and the decomposition of sugary and zero-calorie sales series in the change by the mean level (trend), the periodicity of the data (seasonal variation) and factors that do not show a pattern (random variation).
+
+![**Sale of sugary and zero-calorie drinks by intervention**](../figs/eda-1_time-serie.png){#fig-1}
+
+In particular, @fig-1 shows some important characteristics of the dataset. Firstly, the measurements for each site began at different times. Secondly, the third site experienced a significant increase in sales during most of the first intervention, but afterwards, sales remained at a lower and more stable level. Thirdly, the order of the three calorie messaging interventions was different for each site. Lastly, it is evident from the data that there is a weekly seasonal effect.
+
+# Formal analysis
+
+To evaluate the research questions, it is suggested to utilize a preparation step along with two statistical methods. In the following subsections, the necessary data adjustments, a detailed explanation of each recommended model and the suggested procedure to conduct and analyze the results are presented.
+
+## Pre-analysis
+
+Before conducting the statistical analysis, it is important to address some elements that were identified during the Exploratory Analysis that could impact the results. These include the weekly effect, different lengths of the pre-intervention observations and unnecessary information. To focus on capturing the intervention effects instead of the number of days effects, one can eliminate the weekly impact by either using a percentage of sales or decomposing the time series, as suggested by @ChatfieldXing:2019. While the percentage approach may be easier, it is recommended to use the decomposition method as the variable containing the total items sold may have inconsistencies that suggest unreliable information.[^2]
+
+[^2]: For example, day 199 in the C site has more zero-calorie and sugary beverages sold than total items. Also, days 50 to 53, 107, 184 and 196 in the same site have decimals.
+
+As sites B and C have a short pre-intervention period, it is necessary to compensate for the missing information. This can be done by combining the follow-up and pre-intervention categories into a new category that will serve as the baseline for the study. Lastly, we are not interested in the washout periods that are used to reduce the effects of previous interventions. Hence, this data can be ignored.
```

Comment:
There is no harm to incorporate washout periods in the data as well. These data tells valuable information on how long the effect of intervention will last (although just by looking at the data, the effect only last during the intervention).